CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/DatabaseConfigurationRequestHandler.java

```diff
@@ -10,14 +11,19 @@
     private static final String TAG = "DATABASE_CONFIG";
     public DatabaseConfiguration configure(Args args) {
         String directory = args.get("directory");
-        Log.i(TAG, "database_create name=" + directory);
+        Log.i(TAG, "DatabaseConfiguration_configure directory=" + directory);
         EncryptionKey encryptionKey;
         //ConflictResolver conflictResolver = args.get("conflictResolver");
         String password = args.get("password");
         DatabaseConfiguration config = new DatabaseConfiguration();
         if (directory != null) {
             config.setDirectory(directory);
         }
+        else{
```

Comment:
Since you are setting the variable "directory", why are you calling config.setDirectory twice?